EventSite/Domain/Model/User.cs

```diff
@@ -81,9 +81,14 @@ public class UserProfile {
             public string Company { get; set; }
             public string Bio { get; set; }
             public string TelephoneNumber { get; set; }
+            public bool IsKeynoteSpeaker { get; set; }
             //public string BlogUrl { get; set; }
             //public string TwitterHandle { get; set; }
             //public bool IsMVP { get; set; }
+            public UserProfile()
+            {
+                IsKeynoteSpeaker = false;
```

Comment:
Do we really need this?  The default value for the boolean field should be false without having to explicitly set it. Also we should drop the commented out lines above.